address-export.py

```diff
@@ -69,11 +132,11 @@ def pull_zip(address):
 address_book['zipcode'] = address_book['locality'].apply(pull_zip)
 address_book['town'] = address_book['locality'].map(lambda x: x.split(',')[0])
 
-columnsTitles = ['name', 'address', 'town', 'state','zipcode']
+columnsTitles = ['Name', 'Address', 'Town', 'State','Zipcode']
 
 address_book = address_book.reindex(columns=columnsTitles)
```

Comment:
Line 137 is redundant right now and it was causing me issues, because when you change the columnTitles in line 135 the reindexing zeroes out all the data (existing address_book column names are all lower case).

Can you delete line 137? Otherwise this is fine - I'm agnostic about the column name formatting.